CovertInfiltration/Localization/CovertInfiltration.int

```diff
@@ -140,196 +140,184 @@ strOverviewHeader=INFILTRATION
 strOverviewHeader=ASSAULT
 
 [UIListener_AvengerHUD]
-ChainsOverview_Label=Activity chains
+ChainsOverview_Label=Activity Chains
 
 [UIChainsOverview]
 strOngoing=ONGOING
 strEnded=ENDED
 
 [UIChainsOverview_Activity]
-strCompletionStatusLabel_NotReached=Not reached
-strCompletionStatusLabel_NotCompleted=Not completed yet
+strCompletionStatusLabel_NotReached=Not Reached
+strCompletionStatusLabel_NotCompleted=Not Completed

```

Comment:
"Yet" is a very important part of the logic. "Not Completed" can mean that you skipped the activity, but that's another status. `Not completed yet` means that the activity is still available or ongoing